common/build/eslint-config-fluid/package.json

```
@@ -59,5 +59,13 @@
 		"sort-json": "^2.0.1",
 		"typescript": "~5.1.6"
 	},
-	"packageManager": "pnpm@9.15.3+sha512.1f79bc245a66eb0b07c5d4d83131240774642caaa86ef7d0434ab47c0d16f66b04e21e0c086eb61e62c77efc4d7f7ec071afad3796af64892fae66509173893a"
+	"packageManager": "pnpm@9.15.3+sha512.1f79bc245a66eb0b07c5d4d83131240774642caaa86ef7d0434ab47c0d16f66b04e21e0c086eb61e62c77efc4d7f7ec071afad3796af64892fae66509173893a",
+	"pnpm": {
+		"commentsOverrides": [
+			"serialize-javascript - CVE-2024-11831 comes from mocha 10.4.0, which in turn comes from mocha-multi-reporters 1.5.1 (which has no updated version at this time)"
+		],
+		"overrides": {
+			"serialize-javascript": "^6.0.2"
```

We don't want to accidently replace other versions (maybe someday a 7.0) with 6, so we should be more specific with this override. We can also specifically call out mocha here, so if it stops just being mocha pulling this in, we will get another security error detection indicating that this, and the comment above, need updating, which will prevent the comment from getting out dated.
```suggestion
			"mocha>serialize-javascript@6.0.0": "^6.0.2"
```